agenda-webapps/src/main/webapp/vue-app/agenda-common/components/event/date-poll/AgendaEventDatePollDetails.vue

```diff
@@ -410,12 +410,19 @@ export default {
     sendVotes() {
       const eventId = this.event.id;
       const promises = [];
+      const dateList = [];
       this.sendingVotes = true;
       for(const index in this.dateOptions) {
         const dateOptionId = this.dateOptions[index].id;
         if (this.currentUserVotes.dateOptionVotes[index]) {
-          promises.push(this.$eventService.voteEventDate(eventId, dateOptionId));
+          dateList.push(dateOptionId);
+        }
+      }
+      for(const index in this.dateOptions) {
+        if (this.currentUserVotes.dateOptionVotes[index]) {
+          promises.push(this.$eventService.saveEventVotes(eventId, dateList));
```

Comment:
no neeed to `promises`, you will need to do a single request to server with the list of accepted dateOptions only

Reply:
for the dismiss we send only the dateOptionId so we need promises